src/components/sidebar/Sidebar.jsx

```diff
@@ -0,0 +1,362 @@
+import {useState} from 'react';
+import { Link, NavLink } from 'react-router-dom';
+
+import styled from 'styled-components';
+import { ProjectOutlined } from '@ant-design/icons'
```

Comment:
replace for <Icon..> with the right svg

Reply:
this supposed to be the selected background of the user.. so for now i used some random icon.. we will need to replace it after we implement the backgrounds